library/aesce.h

```diff
@@ -53,6 +53,21 @@ extern "C" {
 int mbedtls_aesce_has_support( void );
 
 
+/**
+ * \brief           Internal round key inversion. This function computes
+ *                  decryption round keys from the encryption round keys.
+ *
+ * \note            This function is only for internal use by other library
+ *                  functions; you must not call it directly.
```

Comment:
This is true for every function in `library/`, only functions declared in `include/` are part of the public API.